### PR TITLE
feat: support config and deploy openmldb-kafka-connect in docker and k8s

### PR DESCRIPTION
**Pull request by tobegit3hub**
* add dockerfile
* add k8s deployment yaml

### CODE REVIEW

**Comment by tobegit3hub** [comment]
connect-standalone.properties: 

```
 # These are defaults. This file just demonstrates how to override some settings.
 bootstrap.servers=10.120.2.40:9092

 # The converters specify the format of data in Kafka and how to translate it into Connect data. Every Connect user will
 # need to configure these based on the format they want their data in when loaded from or stored into Kafka
 key.converter=org.apache.kafka.connect.json.JsonConverter
 value.converter=org.apache.kafka.connect.json.JsonConverter
 # Converter-specific settings can be passed in by prefixing the Converter's setting with the converter we want to apply
 # it to
 key.converter.schemas.enable=false
 value.converter.schemas.enable=true

 offset.storage.file.filename=/tmp/connect.offsets
 # Flush much faster than normal, which is useful for testing/debugging
 offset.flush.interval.ms=10000

 # Set to a list of filesystem paths separated by commas (,) to enable class loading isolation for plugins
 # (connectors, converters, transformations). The list should consist of top level directories that include
 # any combination of:
 # a) directories immediately containing jars with plugins and their dependencies
 # b) uber-jars with plugins and their dependencies
 # c) directories immediately containing the package directory structure of classes of plugins and their dependencies
 # Note: symlinks will be followed to discover dependencies or plugins.
 # Examples:
 # plugin.path=/usr/local/share/java,/usr/local/share/kafka/plugins,/opt/connectors,
 #plugin.path=/usr/local/share/java
 #plugin.path=/Users/tobe/code/4pd/OpenMLDB/extensions/kafka-connect-jdbc/tobedemo/kafka-connect-jdbc
 #plugin.path=/Users/tobe/temp/kafka_plugin_path
 #plugin.path=/Users/tobe/temp/kafka_plugin_path
 #plugin.path=/Users/tobe/code/4pd/openMLDB/extensions/kafka-connect-jdbc/docker
 plugin.path=/tmp



 topic.tracking.enable=true
```

openmldb-sink.properties:

```
 name=test-sink
 connector.class=io.confluent.connect.jdbc.JdbcSinkConnector
 tasks.max=1
 topics=topic1
 connection.url=jdbc:openmldb:///kafka_test?zk=10.120.2.40:2181&zkPath=/openmldb
 auto.create=true
```

K8S config map openmldb-kafka-connect-configmap:

```
kubectl create configmap openmldb-kafka-connect-configmap --from-file=connect-standalone.properties=connect-standalone.properties --from-file=openmldb-sink.properties=openmldb-sink.properties
```

Build docker image command: 

```
docker build -t registry.cn-shenzhen.aliyuncs.com/tobe43/openmldb-kafka-connect -f Dockerfile .
```

Submit K8S deployment command:

```
kubectl create -f ./openmldb-kafka-connect-deployment.yaml
```

File: extensions/kafka-connect-jdbc/docker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM 4pdosc/openmldb:0.7.3
```
Comment (tobegit3hub) [inline extensions/kafka-connect-jdbc/docker/Dockerfile]: consider change to jdk base image.

File: docs/zh/integration/online_datasources/kafka_connector_demo.md [hunk 2]
```diff
@@ -197,3 +197,32 @@ rm -rf /tmp/kafka-logs /tmp/kraft-combined-logs
...
+
+```
+docker build -t registry.cn-shenzhen.aliyuncs.com/tobe43/openmldb-kafka-connect -f Dockerfile .
+```
+
+使用 OpenMLDB 提供的 Yaml 文件来创建 Deployment，可按需修改启动的配置仓库。
+
+```
+kubectl create -f ./openmldb-kafka-connect-deployment.yaml
+```
```
Comment (tobegit3hub) [inline docs/zh/integration/online_datasources/kafka_connector_demo.md:226]: notice that the replica should be 1.

File: docs/zh/integration/online_datasources/kafka_connector_demo.md [hunk 3]
```diff
@@ -197,3 +197,32 @@ rm -rf /tmp/kafka-logs /tmp/kraft-combined-logs
...
+
+部署 Deployment 后，可以查看对应 Pod 的运行状态和日志，然后通过发送消息给 Kafka 检查服务功能是否正常。
```
Comment (tobegit3hub) [inline docs/zh/integration/online_datasources/kafka_connector_demo.md]: Add the service.yaml to expose NodePort port for RESTful.

**Review by dl239 — APPROVED** [review]

**Review by vagetablechicken — APPROVED** [review]

**Pull request merged**